docker-compose.yml

```diff
@@ -359,21 +359,22 @@ services:
 
   model-plugin-deployer:
     image: grapl/model-plugin-deployer:${TAG:-latest}
-    command: |
-      /bin/sh -c '
-        . venv/bin/activate &&
-        cd /home/grapl/app &&
+    entrypoint: ["/bin/bash", "-o", "errexit", "-o", "nounset", "-c"]
```

Comment:
Would it be better to specify the entrypoint in the image?  (This should probably be the entrypoint for everything so we can stop doing && stuff)

Reply:
As for baking this into the images: that's not actually a blanket assumption that we can (or even should) make; it all depends on the nature of the container. The fact that we're having to do all this "extra" work in docker-compose is a bit unique, and not something I'd want to enshrine in our containers.

As for the `&&` stuff, yes, this should ultimately be the entrypoint we use in our docker-compose services that are currently doing `/bin/bash -c 'blah && blah && blah`. I'm updating that as I'm going along with the various services I'm updating.